### PR TITLE
Fix get posts

### PR DESCRIPTION
**Pull request by jarushford**
This PR references issue: We never really made an issue for this...

Brief description of PR: Fixes the table issue we were having earlier today and creates a post template with the data we have access to right now. Also moved the number of following from the main profile page to the list of following, which is set up to pull the number from the data length, we'll just need to change the data source when we can set up that fetch. Also changed some of the cells to just a normal UICell not the dequeueReusableCell method. Not sure I got all of them, so we'll keep an eye on that. This way doesn't reuse cells which isn't quite as efficient, but I had some trouble figuring out resetting cells when the were reused and this was a quicker fix for now, and for the scale of our app.

Specific areas you would like the reviewer to focus on?: Thoughts on the UI of the post? Ideas on how to display the meal and weight/reps info once we have it?

### CODE REVIEW

File: BFit/Controller/FollowingViewController.swift [hunk 1]
```diff
@@ -30,11 +31,12 @@ class FollowingViewController: UIViewController, UITableViewDataSource, UITableV
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        numFollowing.text = "\(mockData.list.count) Following"
         return mockData.list.count
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as UITableViewCell
+        let cell = UITableViewCell.init()
```
Comment (Cody-Price) [inline BFit/Controller/FollowingViewController.swift:39]: Good call, if our app needs to scale we can research this more but for the current scale this is totally fine.

File: BFit/Controller/ProfileViewController.swift [hunk 2]
```diff
@@ -57,8 +59,7 @@ class ProfileViewController: UIViewController, UIImagePickerControllerDelegate,
             response in
             if response.result.isSuccess {
                 self.postsData = JSON(response.data!)["posts"]
-                print(self.postsData)
-//                self.postsTable.performSelector(onMainThread: #selector(UICollectionView.reloadData), with: nil, waitUntilDone: true)
+                self.postsTable.performSelector(onMainThread: #selector(UICollectionView.reloadData), with: nil, waitUntilDone: true)
```
Comment (Cody-Price) [inline BFit/Controller/ProfileViewController.swift:62]: this "waitUntilDone" I'm assuming is swift's version of async?

**Comment by Cody-Price** [comment]
Great job jamie, we can use what we learned here to initialize and manipulate our feed.

File: BFit/Controller/ProfileViewController.swift (hunk 2 above)
Comment (jarushford) [reply BFit/Controller/ProfileViewController.swift:62]: Yeah, it's one way of doing it I believe

**Pull request merged**